src/issueModal/sass/_dismiss-panel.scss

```
@@ -0,0 +1,104 @@
+	list-style: none;
+
+	dt {
+		font-weight: 600;
+	}
+
+	dd {
+		font-size: 13px;
+		color: #1e1e1e;
+		margin: 0;
+		padding: 0;
+		word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)